test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -29,7 +29,7 @@
 
 (deftest whitelisting
   (testing "should log on reject"
-    (let [wl (fs/temp-file)]
+    (let [wl (fs/temp-file "whitelist-log-reject")]
       (.deleteOnExit wl)
```

Comment:
Note that this approach will create temp names like whitelist-log-reject1232135...  I could imagine being fine with that, specifying (temp-dir "foo-") instead, or arguing that a temp-file wrapper should handle it.